core/frontend/src/WebViewerApp.ts

```diff
@@ -41,8 +42,15 @@ export class WebViewerApp {
       BentleyCloudRpcManager.initializeClient(params.rpcParams, opts.iModelApp?.rpcInterfaces ?? [], params.routing);
     }
     await IModelApp.startup(opts.iModelApp);
-    if (opts.webViewerApp.authConfig)
-      IModelApp.authorizationClient = new BrowserAuthorizationClient(opts.webViewerApp.authConfig);
+
+    if (opts.webViewerApp.authConfig) {
+      await BrowserAuthorizationCallbackHandler.handleSigninCallback(opts.webViewerApp.authConfig.redirectUri);
+      const auth = new BrowserAuthorizationClient(opts.webViewerApp.authConfig);
+      IModelApp.authorizationClient = auth;
+      try {
+        await auth.signInSilent(new ClientRequestContext());

```

Comment:
Can you add this to the NextVersion.md? This is an important change as all apps today are calling this and now they're going to be doing the same thing twice without changing their code.

Reply:
it should be in NextVersion, but apps won't be affected unless they change their code . This is a method on `WebViewerApp`, which we just switched ui-test-app to use. Nobody else is using that, since it's new.

Reply:
Ah true, missed that part initially.